crates/gitbutler-tauri/Cargo.toml

```diff
@@ -37,11 +38,19 @@ once_cell = "1.20"
 reqwest = { version = "0.12.8", features = ["json"] }
 serde.workspace = true
 serde_json = { version = "1.0", features = ["std", "arbitrary_precision"] }
-tauri-plugin-context-menu = { git = "https://github.com/c2r0b/tauri-plugin-context-menu", branch = "main" }
-tauri-plugin-single-instance = { git = "https://github.com/tauri-apps/plugins-workspace", branch = "v1" }
-tauri-plugin-window-state = { git = "https://github.com/tauri-apps/plugins-workspace", branch = "v1" }
-tauri-plugin-store = { git = "https://github.com/tauri-apps/plugins-workspace", branch = "v1" }
-tauri-plugin-log = { git = "https://github.com/tauri-apps/plugins-workspace", branch = "v1" }
+tauri = { version = "^2.0.6", features = ["unstable"] }
+tauri-plugin-clipboard-manager = "2.0.1"
```

Comment:
where is the clipboard-manager used?

Reply:
nice catch. looks like it's not used anywhere, removing it.